src/concrete/ml/quantization/quantized_module.py

```diff
@@ -388,19 +394,22 @@ def quantized_forward(
         )
 
         if fhe == "disable":
-            return self._clear_forward(*q_x)
-
-        simulate = fhe == "simulate"
-        return self._fhe_forward(*q_x, simulate=simulate)
+            result = self._clear_forward(*q_x)
```

Comment:
not sure I understand this change ?

Reply:
Ho yeah it was just for debugging purposes, I'll revert that, thanks for noticing